spec/e2e/query_spec.rb

```diff
@@ -86,9 +86,9 @@ def self.ordered_by_subject
       student.lessons << lesson
 
       expect(Student.as(:s).pluck(:s)).to eq([student])
-      expect(Student.all.pluck(:uuid)).to eq([student.uuid])
+      expect(Student.all.pluck(:id)).to eq([student.id])
```

Comment:
Here `pluck(:id)` is working as `all` returns object of class implemented in neo4j.